chore: Add initial V9 codemod (#2021)
Workday/canvas-kit: modules/docs/mdx/MAINTAINING.mdx

`build-number` is the GitHub build identifier, and `commit-count` is the number of commits since the
last release tag. So for example, a V8 canary would look something like this:
`8.0.0-alpha.127-next.4`.

## Codemods

> We use codemods to reduce friction for consumers as they make changes and upgrade. Codemods are

Reviewer: ```suggestion
We use codemods to reduce friction for consumers as they make changes and do upgrades. Codemods are
```

> often accompany major version releases, but not exclusively.

Reviewer: ```suggestion
accompany major version releases since v5, and can also be released in minor releases if users want to apply some changes sooner.
```

### Add a New Codemod

> Adding a new codemod is pretty straightforward, but guide will make sure you don't miss any steps

Reviewer: ```suggestion
Adding a new codemod is pretty straightforward, but this guide will make sure you don't miss any steps
```
